05.ls/ls_command.rb

```diff
@@ -44,20 +44,20 @@ def make_l_option_file
 end
 
 def show_l_option(file_data_list)
-  max_nlink_size = file_data_list.max_by { |file_data| file_data[:nlink].size }[:nlink].to_s.size + 2
+  max_nlink_size = file_data_list.max_by { |file_data| file_data[:nlink].to_s.size }[:nlink].to_s.size
```

Comment:
```suggestion
  max_nlink_size = file_data_list.map { |file_data| file_data[:nlink].to_s.size }.max
```

こういう書き方もありますが、mapとmaxでループが2回走るので処理効率はちょっとよくないかも（ただの推測ですが）。
というわけで、今のままでもOKです。